tests_with_kvdb.sh

```diff
@@ -17,12 +17,6 @@ if [ "$1" = "dev" ]
 then
   echo "Executing only 'dev' tests ..."
   pytest -m "dev"
-elif [ "$1" = "integrationtest" ]
```

Comment:
See above. If kept as `integrationtests`, than keep also within this script and folder structure (i.e. `tests/integrationtests/`, instead of just `tests/`